ImageButton should be with srcCompat instead of src

`srcCompat` should be put in all 3 layout files.

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.buildware.geoworld/com.nononsenseapps.filepicker.FilePickerActivity}: android.view.InflateException: Binary XML file line #89: Binary XML file line #89: Error inflating class ImageButton
                                                                at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2646)
                                                                at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
                                                                at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                                at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
                                                                at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                at android.os.Looper.loop(Looper.java:154)
                                                                at android.app.ActivityThread.main(ActivityThread.java:6077)
                                                                at java.lang.reflect.Method.invoke(Native Method)
                                                                at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                                                                at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
                                                             Caused by: android.view.InflateException: Binary XML file line #89: Binary XML file line #89: Error inflating class ImageButton
                                                             Caused by: android.view.InflateException: Binary XML file line #89: Error inflating class ImageButton
                                                             Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 5: TypedValue{t=0x2/d=0x7f010020 a=-1}
                                                                at android.content.res.TypedArray.getColorStateList(TypedArray.java:528)
                                                                at android.widget.ImageView.<init>(ImageView.java:175)
                                                                at android.widget.ImageButton.<init>(ImageButton.java:84)
                                                                at android.widget.ImageButton.<init>(ImageButton.java:80)
                                                                at android.support.v7.widget.AppCompatImageButton.<init>(AppCompatImageButton.java:57)
                                                                at android.support.v7.widget.AppCompatImageButton.<init>(AppCompatImageButton.java:53)
                                                                at android.support.v7.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:118)
                                                                at android.support.v7.app.AppCompatDelegateImplV7.createView(AppCompatDelegateImplV7.java:1008)
                                                                at android.support.v7.app.AppCompatDelegateImplV7.onCreateView(AppCompatDelegateImplV7.java:1067)
                                                                at android.support.v4.view.LayoutInflaterCompatHC$FactoryWrapperHC.onCreateView(LayoutInflaterCompatHC.java:44)
                                                                at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:192)
                                                                at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:769)
                                                                at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:727)
                                                                at android.view.LayoutInflater.rInflate(LayoutInflater.java:858)
                                                                at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
                                                                at android.view.LayoutInflater.rInflate(LayoutInflater.java:861)
                                                                at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
                                                                at android.view.LayoutInflater.rInflate(LayoutInflater.java:861)
                                                                at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
                                                                at android.view.LayoutInflater.inflate(LayoutInflater.java:518)
                                                                at android.view.LayoutInflater.inflate(LayoutInflater.java:426)
                                                                at com.nononsenseapps.filepicker.AbstractFilePickerFragment.onCreateView(AbstractFilePickerFragment.java:163)
                                                                at android.support.v4.app.Fragment.performCreateView(Fragment.java:2074)
                                                                at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1104)
                                                                at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1286)
                                                                at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:758)
                                                                at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1671)
                                                                at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:388)
                                                                at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:619)
                                                                at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1248)
                                                                at android.app.Activity.performStart(Activity.java:6681)
                                                                at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2609)
                                                                at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
                                                                at android.app.ActivityThread.-wrap12(ActivityThread.java)
09-14 20:09:44.348 2367-2367/com.buildware.geoworld E/ACRA:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
                                                                at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                at android.os.Looper.loop(Looper.java:154)
                                                                at android.app.ActivityThread.main(ActivityThread.java:6077)
                                                                at java.lang.reflect.Method.invoke(Native Method)
                                                                at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                                                                at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

I'm pretty sure the recommended way is to replace `src` with `srcCompat` when you have an `AppCompat'-based theme and a vector drawable.

--- a/examples/build.gradle
+++ b/examples/build.gradle
@@ -10,6 +10,7 @@ android {
         targetSdkVersion 24
         versionCode 1
         versionName "1.0"
+        vectorDrawables.useSupportLibrary = true
     }
     buildTypes {
         release {

--- a/library/build.gradle
+++ b/library/build.gradle
@@ -22,6 +22,7 @@ android {
         targetSdkVersion 23
         versionCode gitCommitCount
         versionName gitTag
+        vectorDrawables.useSupportLibrary = true
     }
 }
 

--- a/library/src/main/res/layout/nnf_filepicker_listitem_checkable.xml
+++ b/library/src/main/res/layout/nnf_filepicker_listitem_checkable.xml
@@ -6,6 +6,7 @@
   ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
   -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/nnf_item_container"
     android:layout_width="match_parent"
@@ -25,7 +26,7 @@
         android:layout_height="?android:listPreferredItemHeight"
         android:adjustViewBounds="true"
         android:scaleType="center"
-        android:src="@drawable/nnf_ic_folder_black_48dp"
+        app:srcCompat="@drawable/nnf_ic_folder_black_48dp"
         android:tint="?attr/nnf_dir_icon_color"
         android:visibility="visible"
         tools:ignore="ContentDescription" />

--- a/library/src/main/res/layout/nnf_filepicker_listitem_dir.xml
+++ b/library/src/main/res/layout/nnf_filepicker_listitem_dir.xml
@@ -6,6 +6,7 @@
   ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
   -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/nnf_item_container"
     android:layout_width="match_parent"
@@ -24,7 +25,7 @@
         android:layout_height="?android:listPreferredItemHeight"
         android:adjustViewBounds="true"
         android:scaleType="center"
-        android:src="@drawable/nnf_ic_folder_black_48dp"
+        app:srcCompat="@drawable/nnf_ic_folder_black_48dp"
         android:tint="?attr/nnf_dir_icon_color"
         android:visibility="visible"
         tools:ignore="ContentDescription" />

--- a/library/src/main/res/layout/nnf_fragment_filepicker.xml
+++ b/library/src/main/res/layout/nnf_fragment_filepicker.xml
@@ -10,6 +10,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     tools:context=".FilePickerActivity">
 
     <android.support.v7.widget.Toolbar
@@ -92,7 +93,7 @@
                 android:layout_width="48dp"
                 android:layout_height="48dp"
                 android:hint="@android:string/ok"
-                android:src="@drawable/nnf_ic_save_black_24dp"
+                app:srcCompat="@drawable/nnf_ic_save_black_24dp"
                 android:tint="?attr/nnf_save_icon_color"/>
 
         </LinearLayout>

--- a/sample/build.gradle
+++ b/sample/build.gradle
@@ -30,6 +30,7 @@ android {
         versionName gitTag
         archivesBaseName = "nononsensefilepicker-sample-${gitTag}".toString()
         testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+        vectorDrawables.useSupportLibrary = true
     }
 
     buildTypes {

--- a/sample/src/main/res/layout/fragment_fastscrollerfilepicker.xml
+++ b/sample/src/main/res/layout/fragment_fastscrollerfilepicker.xml
@@ -96,7 +96,7 @@
                 android:layout_width="48dp"
                 android:layout_height="48dp"
                 android:hint="@android:string/ok"
-                android:src="@drawable/nnf_ic_save_black_24dp"
+                app:srcCompat="@drawable/nnf_ic_save_black_24dp"
                 android:tint="?attr/nnf_save_icon_color"/>
 
         </LinearLayout>

--- a/sample/src/main/res/layout/fragment_loading_filepicker.xml
+++ b/sample/src/main/res/layout/fragment_loading_filepicker.xml
@@ -9,6 +9,7 @@
 <!-- This file is just a copy of the normal nnf_fragment_filepicker, with an added progress bar -->
 
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
@@ -103,7 +104,7 @@
                 android:layout_width="48dp"
                 android:layout_height="48dp"
                 android:hint="@android:string/ok"
-                android:src="@drawable/nnf_ic_save_black_24dp"
+                app:srcCompat="@drawable/nnf_ic_save_black_24dp"
                 android:tint="?attr/nnf_save_icon_color"/>
 
         </LinearLayout>
@@ -113,6 +114,6 @@
         android:id="@+id/divider"
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:layout_above="@id/nnf_button_container"
+        android:layout_above="@id/nnf_buttons_container"
         android:background="?nnf_separator_color"/>
 </RelativeLayout>

